playground/backend/cmd/server/controller.go

```diff
@@ -193,7 +196,7 @@ func (controller *playgroundController) Cancel(ctx context.Context, info *pb.Can
 		return nil, errors.InvalidArgumentError("Cancel", "pipelineId has incorrect value and couldn't be parsed as uuid value: %s", info.PipelineUuid)
 	}
 	if err := utils.SetToCache(ctx, controller.cacheService, pipelineId, cache.Canceled, true); err != nil {
-		return nil, errors.InternalError("Cancel", "error during set cancel flag to cache")
+		return nil, errors.InternalError("Cancel", "Error during set cancel flag to cache as a true")
```

Comment:
"as a true" is redundant (as well as "as a false" above)

Reply:
Done.